### PR TITLE
(GH-2013) Update docs on PowerShell execution policy

### PR DESCRIPTION
**Pull request by beechtom**
This updates our documentation on PowerShell execution policy when
installing Bolt on Windows. Since Bolt's PowerShell module is now
signed, users on Bolt >= 2.21.0 will likely not need to change their
script execution policy to use Bolt.

!no-release-note

**Note:** The text for this is taken directly from [James' comment](https://github.com/puppetlabs/bolt/issues/2013#issuecomment-675027455) in the issue.

### CODE REVIEW

**Review by lucywyman — APPROVED** [review]
LGTM! I'll wait for review from @hestonhoffman and @jpogran .

**Review by jpogran — APPROVED** [review]
LGTM!

File: documentation/bolt_installing.md
Comment (hestonhoffman) [inline documentation/bolt_installing.md:119]:
```suggestion
As of Bolt 2.21.0, we sign Bolt PowerShell module files with a Puppet code
signing certificate. If your PowerShell environment uses an `AllSigned` execution 
policy and you add Puppet as a trusted publisher, the `bolt` command works 
without any further input. If you're using the `AllSigned` policy and you have 
not added Puppet as a trusted publisher, you can accept the publisher without 
having to change your execution policy.

If your environment uses a `Restricted` policy, you must change your policy to 
`RemoteSigned` or`AllSigned`. Check with your security team before you make 
any policy changes.
```

File: documentation/bolt_installing.md
Comment (hestonhoffman) [inline documentation/bolt_installing.md:122]:
```suggestion
If you see this or a similar error when trying to run Bolt, you probably need to
change your script execution policy restrictions:
```

File: documentation/bolt_installing.md
Comment (hestonhoffman) [inline documentation/bolt_installing.md:134]:
```suggestion
To change your script execution policy:
```

File: documentation/bolt_installing.md
Comment (hestonhoffman) [reply documentation/bolt_installing.md:119]: Question: How would one add Puppet as a trusted publisher? Talk to your security team?

Comment (beechtom) [reply documentation/bolt_installing.md:119]: My understanding is that you're considered a trusted publisher if the script is signed with a valid, unexpired digital signature issued by a certificate authority. @jpogran would know better since he's much more familiar with Windows, but I don't think we need to cover that since it only applies to users running old versions of Bolt.

Comment (hestonhoffman) [reply documentation/bolt_installing.md:119]: @beechtom After consulting with James, I don't think we need to add anything else to this edit 👍

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**
